package.json

```diff
@@ -7,6 +7,7 @@
     "@ensdomains/buffer": "^0.0.6",
     "@ensdomains/solsha1": "^0.0.2",
     "dnsprovejs": "0.0.9",
+    "elliptic-solidity": "git+https://github.com/tdrerup/elliptic-curve-solidity.git",
```

Comment:
Is it possible to pin this to a commit hash? I'd rather not have a dependency on solidity code that can be changed without notice.